oak_runtime/build.rs

```diff
@@ -37,4 +38,30 @@ fn main() {
             ..Default::default()
         },
     );
+
+    // Copy introspection_browser_client files (generated by the runner) to the
+    // out dir. In case these files don't yet exist, such as when running unit
+    // tests, create placeholder files. This ensures unit tests etc can be run
+    // without needing to build the frontend client assets.
+    const INTROPSECTION_CLIENT_DIR: &str = "src/introspection_browser_client/dist";
+    const INTROPSECTION_CLIENT_FILES: &[&str] = &["index.html", "index.js"];
+    let out_dir = env::var("OUT_DIR").unwrap();
+    fs::create_dir_all(&format!("{}/{}", out_dir, INTROPSECTION_CLIENT_DIR))
+        .expect("unable to create introspection client out directory");
+    for client_file in INTROPSECTION_CLIENT_FILES {
+        let path = format!("{}/{}", INTROPSECTION_CLIENT_DIR, client_file);
+        let out_path = format!("{}/{}", out_dir, path);
+        let mut out_file = fs::OpenOptions::new()
+            .create(true)
+            .write(true)
+            .truncate(true)
+            .open(&out_path)
+            .expect("unable to open/create data file");
+        if let Ok(mut source_file) = fs::File::open(&path) {
+            io::copy(&mut source_file, &mut out_file).expect("failed to copy data after opening");
+        }
```

Comment:
This seems a bit risky, it will silently use the placeholders if the upstream steps failed / were missed. I think a config feature flag to use real or mock files would be more explicit.

Reply:
Makes sense. By a config flag to you mean a cargo feature flag? I would like to try that idea, it might even eliminate the need for these build rules.

However I'm having trouble setting my own flags, despite adding them to Cargo.toml & passing them via `--features` to cargo in the runner.